src/sx/README.md

```diff
@@ -142,6 +142,38 @@ const styles = sx.create({
 });
 ```
 
+Note that if you want to add some styling of the same property with different pseudo classes, there might be some specificity issues. Say you want to do this:
+
+```js
+const styles = sx.create({
+  button: {
+    ':hover': {
+      color: 'pink',
+    },
+    ':active': {
+      color: 'blue',
+    },
+  },
+});
+```
+
+These 2 rules will have the same specificity, and the one defined last in the stylesheet will win. It may or may not help to change order in this style, because the class could be created by a different rule. What will help is to raise the specificity of the active class. You can do this:
```

Comment:
Do you have a real-world example? Maybe some failing fixture? I am not sure I 100% understand the problem from the description.

Reply:
Check [this codepen](https://codesandbox.io/s/crimson-worker-58t6m?file=/src/styles.css)

Clicking the button doesn't change the color. Then uncomment the commented CSS, and the the color changes when you click the button.

Reply:
Or [this](https://codesandbox.io/s/clever-ardinghelli-n0ry8?file=/src/index.js), since it uses `sx`